From 178a799319379d690c080c6d354a58cab6979662 Mon Sep 17 00:00:00 2001
From: Richard W.M. Jones <rjones@redhat.com>
Date: Thu, 21 Nov 2013 13:57:07 +0000
Subject: [PATCH 29/67] builder/sysprep: Describe --password/--root-password option as "selector".

It now appears in the respective man pages as:

  --root-password selector

or:

  --password selector

This avoids confusion from people who think these command line options
take the password directly.

(cherry picked from commit 9186438937f5451ddfdaa33c9dcb9ffe27b2a098)
---
 sysprep/sysprep_operation_delete.ml |    8 +++++---
 1 files changed, 5 insertions(+), 3 deletions(-)

diff --git a/sysprep/sysprep_operation_delete.ml b/sysprep/sysprep_operation_delete.ml
index 7cf1c39..de78a87 100644
--- a/sysprep/sysprep_operation_delete.ml
+++ b/sysprep/sysprep_operation_delete.ml
@@ -40,9 +40,11 @@ Delete specified files or directories.
 
 Use the I<--delete> option to specify a path to remove.");
     extra_args = [
-      ("--delete", Arg.String add_paths, s_"path" ^ " " ^ s_"File or directory to be removed on guest"),
-      s_"\
-Delete (recursively) the specified C<path> on guest.";
+      { extra_argspec = ("--delete", Arg.String add_paths, s_"path" ^ " " ^ s_"File or directory to be removed on guest");
+        extra_pod_argval = Some "PATHNAME";
+        extra_pod_description = s_"\
+Delete (recursively) the specified C<PATHNAME> in the guest.";
+      }
     ];
 
     perform_on_filesystems = Some path_perform;
-- 
1.7.1

